Makefile.work

```diff
@@ -200,6 +203,8 @@ SONIC_BUILD_INSTRUCTION :=  make \
                            HTTPS_PROXY=$(https_proxy) \
                            SONIC_ENABLE_SYSTEM_TELEMETRY=$(ENABLE_SYSTEM_TELEMETRY) \
                            SONIC_ENABLE_RESTAPI=$(ENABLE_RESTAPI) \
+                           EXTRA_JESSIE_TARGETS=$(EXTRA_JESSIE_TARGETS) \
```

Comment:
Why do we have this EXTRA_JESSIE_TARGETS ?
Is this a side effect of resolving conflict?

Reply:
Hmm, it seems so. I have removed it now. Thanks.